toonz/sources/tnzbase/tscanner/tscannerutil.cpp

```diff
@@ -2,6 +2,10 @@
 
 #include "tscannerutil.h"
 
+#ifdef LINUX
+#include <cstring>
+#endif
```

Comment:
Removed Linux ifdef's,

Only added them because (I assume), its not needed on other platforms.
However I don't have a strong opinion here.